### DISCUSSION TITLE
Create a dafult walk-by customer

### DISCUSSION THREAD

**Discussion by mrports** [Blair2004/NexoPOS#884] [Ideas]
from my experience as having an agency that provide installation of POS software for many types of stores, i found it common that store owners are mostly concerned about how fast the POS can be used, so their employees can quickly finish the job and let the customer leave as quick as possible especially when the store is crowded thats why i will be offering some suggestions that i hope that will help improve the POS quality
1-let the barcode field always in the auto focus (always ready)
2-give the option in the POS setting to use "walk-by-cutomer" as a default customer or the option to choose from a customer list as it is currently
3-use "ENTER" button to access the payment screen modal
4-make it possible to use the keyboard on the payment screen modal and also possible to use the "ESC" button to hide it
5-adding the option to disable the payment confirmation message

**Comment by Blair2004** [comment]
Hi,
1- that i believe is another thread
2- You can from the settings assign which customer will be the default customer
3- Scheduled for v4.8.x
4- same as 3.
5- ... not sure it's safe.

Thank you for your ideas.
